### PR TITLE
Remove unused EdgeRequestQueue class

### PR DESCRIPTION
**Pull request by addb**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -42,3 +42,12 @@ build-sdk:
 # usage: make check-version VERSION=1.0.0
 check-version:
 	sh ./build/version.sh $(VERSION)
+
+all:
+	@echo "############################### Make all ###############################"
```
Comment (praveek) [inline Makefile:47]: Can you use the debug statements to respective targets and use make target dependency instead? 
`all: install-bsc archive`

File: sample/development/components/Screens/NewScreen.xml [hunk 2]
```diff
@@ -68,7 +68,6 @@ governing permissions and limitations under the License.-->
   <script type="text/brightscript" uri="pkg:/source/adobe/core/identityState.brs"/>
   <script type="text/brightscript" uri="pkg:/source/adobe/edge/edgeModule.brs"/>
   <script type="text/brightscript" uri="pkg:/source/adobe/edge/edgeRequest.brs"/>
```
Comment (praveek) [inline sample/development/components/Screens/NewScreen.xml:70]: I notice adding or removing a file requires changes to lot of other files. Can you add a task to automate this?

File: Makefile [hunk 3]
```diff
@@ -42,3 +42,12 @@ build-sdk:
...
+	@echo "############################### Installing BrighterScript Compiler ###############################"
+	make install-bsc
+	@echo "############################### Building AEP Roku SDK ###############################"
+	make archive
+	@echo "############################### Building Sample app (simple-videoplayer-channel) ###############################"
+	cd ./sample/simple-videoplayer-channel && make install-sdk && make build
```
Comment (praveek) [inline Makefile:53]: Can you do make -C "sub-directory" install-sdk build

File: sample/development/components/Screens/NewScreen.xml (hunk 2 above)
Comment (addb) [reply sample/development/components/Screens/NewScreen.xml:70]: Sure

**Comment by addb** [comment]
@praveek Improved makefile and version.sh script

Screenshot 2024-09-10 at 3 56 36 PM

**Review by praveek — APPROVED** [review]

**Review by praveek — APPROVED** [review]

**Pull request merged**